src/locations/index.ts

```diff
@@ -54,39 +54,64 @@ export async function getLocations(options: LocationCmdOptions) {
 }
 
 export function formatLocations(locations: Array<LocationMetadata>) {
-  const formatted: Array<string> = [];
+  const publicLocations: Array<string> = [];
+  const privateLocations: Array<string> = [];
   for (const location of locations) {
     let name = location.label;
     if (location.isServiceManaged) {
       [, name] = name.includes('-') ? name.split('-') : [, name];
       name = name.toLowerCase().trim().split(' ').join('_');
+      publicLocations.push(name);
     } else {
-      name = `${name}${PRIVATE_KEYWORD}`;
+      privateLocations.push(name);
     }
-    formatted.push(name);
   }
-  return formatted;
+  return {
+    publicLocations,
+    privateLocations,
+    allLocations: [
```

Comment:
use concat.

Reply:
Is using spread operator wrong in this ?

Reply:
unnecessary copy. would like to keep it simpler than using spread when not required.